Modules/mathmodule.c

```diff
@@ -1214,6 +1214,40 @@ math_floor(PyObject *module, PyObject *number)
     return PyLong_FromDouble(floor(x));
 }
 
+/*[clinic input]
+math.fmax -> double
+
+    x: double
+    y: double
+    /
+
+Returns the larger of two floating-point arguments.
```

Comment:
```suggestion
Return the larger of two floating-point arguments.
```

To align with the other descriptions